pl_bolts/datamodules/cityscapes_datamodule.py

```diff
@@ -177,3 +188,10 @@ def default_transforms(self):
             )
         ])
         return cityscapes_transforms
+
+    def default_target_transforms(self):
+        cityscapes_target_trasnforms = transform_lib.Compose([
+            transform_lib.ToTensor(),
+            transform_lib.Lambda(lambda t: t.squeeze())
```

Comment:
I have added this transform to labels as well as limited `target_type` to 'instance' and 'semantic' to ensure out-of-the-box operability with `pl_bolts.models.vision.SemSegment`. I'm wondering wheter it'd be more proper not to transform targets and return what torchvision's original CityScapes dataset returns and drop the requirements of instance/semantic target_type only. In such case, user of this DataModule would be responsible to transform targets to desired format but overall this DataModule would be more flexible